backend/main/src/main/java/sentiment/settings/SetSettingsRequest.java

```diff
@@ -58,4 +67,38 @@ private String setSetting(String name, String value) {
     return "";
 
   }
+
+  /**
+   * Updates the Lambda invocation interval for the given setting, if necessary.
+   * @param settingName The setting name
+   * @param settingValue The setting value
+   */
+  private String updateLambdaInterval(String settingName, String settingValue) {
```

Comment:
Is returning a string just for exceptions?

Reply:
Yep! I've set up the Response classes where they either take a string to return (in an error state) or a result to return.